src/IW/Core/Repo.hs

```diff
@@ -27,7 +27,10 @@ data Repo = Repo
     , repoDescr      :: !Text
     , repoCategories :: !(SqlArray Text)
     } deriving stock (Generic, Show, Eq)
-      deriving anyclass (ToJSON, FromRow, ToRow)
+      deriving anyclass (ToJSON, FromRow)
+
+instance ToRow Repo where
+    toRow Repo{..} = toRow (repoOwner, repoName, repoDescr, repoCategories)
```

Comment:
I think this instance should be derived. Probably this was an issue with resolving conflicts...

Reply:
Oops. Fixed it.